### PR TITLE
Hyperlinks to headers fixed; font changes

### PR DESCRIPTION
**Pull request by kartik-ynwa**
- Hyperlinks to headers were missing. Fixed those; link text is exactly the same as the chapter name.
- Removed remote fonts. This lets the reader read the book in a font that they are comfortable in. Adheres to bettermotherfuckingwebsite.com guidelines.

### CODE REVIEW

File: basics.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ a standalone executable.
 
 `import "fmt"` says we need `fmt` in addition to `main` <2>. A package other
 than `main` is commonly called a library, a familiar concept in many programming
-languages (see (#packages)). The line ends with a comment that begins with `//`.
+languages (see [Packages](#packages)). The line ends with a comment that begins with `//`.
```
Comment (miekg) [inline basics.md]: This should not be needed, `(#packages)` is sufficient. And the the html is work, for mmarks RFC output this should also be enough

**Comment by miekg** [comment]
Can do the css changes - but they need to come in another PR.

**Comment by kartik-ynwa** [comment]
Yeah you're right. I swear to god though when I was reading on my laptop I
didn't see the links. Now I open it on my phone and everything is okay.
Spooky stuff right there.

Sorry for wasting your time.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [miekg/learninggo] Hyperlinks t..." ]
>Yeah you're right. I swear to god though when I was reading on my laptop I
>didn't see the links. Now I open it on my phone and everything is okay.
>Spooky stuff right there.
>
>Sorry for wasting your time.

Np, is javascript inserting these, so.....

**Comment by kartik-ynwa** [comment]
That explains it. It's uMatrix's fault then. Good job detective.

**Pull request closed without merging**